src/app/components/Label.jsx

```diff
@@ -0,0 +1,6 @@
+import styles from "../stylesheet/components/Label.module.css";
+export default function Label(props) {
+  return (
+    <label htmlFor="" id={props.id} className={styles[props.class]}> {props.text}</label>
+  );
+}
```

Comment:
If you're not adding any special behavior or functionality, this is an overkill.